recipes/libv8/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "libv8" %}
+{% set version = "7.5.288.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://github.com/v8/v8/archive/{{ version }}.tar.gz"
+    sha256: df12222ebdb9538f62434785ea246f1d27f198b5d674e0577dd1487ffe53b6e2
+  # We tarballs but don't add sha256s as they are different on every download.
+  # Sychronized hashes from https://github.com/Homebrew/homebrew-core/blob/2a8b9b60c832d25c8b2013d641e49a886ccbe0f7/Formula/v8.rb
+  - url: "https://chromium.googlesource.com/chromium/src/build.git/+archive/a0b2e3b2708bcf81ec00ac1738b586bcc5e04eea.tar.gz"
+    folder: build
+  - url: "https://chromium.googlesource.com/chromium/src/third_party/jinja2.git/+archive/b41863e42637544c2941b574c7877d3e1f663e25.tar.gz"
+    folder: third_party/jinja2
+  - url: "https://chromium.googlesource.com/chromium/src/third_party/markupsafe.git/+archive/8f45f5cfa0009d2a70589bcda0349b8cb2b72783.tar.gz"
+    folder: third_party/markupsafe
+  - url: "https://chromium.googlesource.com/external/github.com/google/googletest.git/+archive/b617b277186e03b1065ac6d43912b1c4147c2982.tar.gz"
+    folder: third_party/googletest/src
+  - url: "https://chromium.googlesource.com/chromium/src/base/trace_event/common.git/+archive/ebb658ab38d1b23183458ed0430f5b11853a25a3.tar.gz"
+    folder: base/trace_event/common
+  - url: "https://chromium.googlesource.com/chromium/deps/icu.git/+archive/35f7e139f33f1ddbfdb68b65dda29aff430c3f6f.tar.gz"
+    folder: third_party/icu
+    patches:
+      - third_party-icu-use-system-icu.diff
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
+    - clangdev  # [osx]
+    - gn
+    - python <3
+    - ninja
+  host:
+    - glib
+    - icu =64.2
+  run:
+    - icu =64.2
```

Comment:
We should bump the pinning to this version.

Reply:
I'll take care of that but I actually want to let the R migration move a bit fowards as `icu` repinnings also affect a lot of R.